File: snippets/php/array/set/set.php
<?php
$arraay = [3, 2, 1];
$index = 1;  // 0: first item, 1: second item...
$item = 100;
var_export($arraay);  // array (0 => 3, 1 => 2, 2 => 1, )
$arraay[$index] = $item;  // ≡
var_export($arraay);  // array (0 => 3, 1 => 100, 2 => 1, )
?>

File: snippets/rs/array/set/set.rs
fn main()
{
    let mut array = vec![3, 2, 1];
    let index = 1;  // 0: first item, 1: second item...
    let item = 100;
    println!("{array:?}");  // [3, 2, 1]
    array[index] = item;  // ≡
    println!("{array:?}");  // [3, 100, 1]
}